data/brands/shop/truck.json

```
@@ -42,6 +42,17 @@
         "shop": "truck"
       }
     },
+    {
+      "displayName": "Peterbilt",
+      "id": "peterbilt-13eeb7",
+      "locationSet": {"include": ["001"]},
```

It looks like most locations are still [tagged as `shop=car`](https://overpass-turbo.eu/s/1AH3). This will match existing locations tagged as `shop=car` with this entry.
```suggestion
      "locationSet": {"include": ["001"]},
      "matchTags" ["shop/car"],
```